dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathEntry.java

```diff
@@ -112,13 +114,15 @@ private void readClassFileNames() throws IOException {
       this.classFileNames = classNames.build();
     }
   }
-  
+
   /**
-   * Returns the fully qualified names of the classes in this entry's jar file.
-   * 
```

Comment:
The previous javadoc was incorrect. It is not fully qualified class names. It returns class file names in a JAR.